server/session/handler_item_stack_request.go

```diff
@@ -269,6 +275,39 @@ func (h *ItemStackRequestHandler) handleMineBlock(a *protocol.MineBlockStackRequ
 	return nil
 }
 
+// handleCreate handles the creation of a pending result through a craft.
```

Comment:
I would document this more thoroughly. What does the `CreateStackRequestAction` do here exactly? I have a bit of a hard time understanding what's happening here.

Reply:
It's a stack request action sent by the client when a recipe outputs more than one item. It contains a result slot, which should map to one of the output items and the server should create the relevant output as usual.

Reply:
You should document that :wink:

Reply:
Updated the documentation there.